event_contact/views/event_event_view.xml

```diff
@@ -2,14 +2,14 @@
 <openerp>
     <data>
 
-        <record model="ir.ui.view" id="event_contacts_form_view">
+        <record model="ir.ui.view" id="event_event_form_view">
```

Comment:
Better not change XMLID if not strictly required. Complicates things for migrations.